### DISCUSSION TITLE
Zero Price for a variant

### DISCUSSION THREAD

**Discussion by maxpanakov** [medusajs/medusa#2419] [Q&A] [answered]
Hey, after updating the admin panel, we can no longer set a zero price, may I ask why?

**Comment by srindom** [comment]
@maxpanakov thanks for reporting this is not intended. @kasperkristensen - maybe the price editor update needs a quick fix here?

**Reply by maxpanakov** [reply]
Is there also a way to remove the price for the region? We somehow set them to zero, but now we want to remove them, and they stay in a variant even after an update

**Reply by maxpanakov** [reply]
And when we tried to duplicate product with region prices, we got an error because in the price there were fields: currency and region.

**Comment by kasperkristensen** [comment, marked as answer]
I just opened a PR that should fix the issue that prevents setting the variant prices to 0 in both the create and edit flows;https://github.com/medusajs/admin/pull/727. 

The issue with region prices not being deleted seems to be a backend issue, that I will look into tomorrow :)
